packages/vue-client/src/components/GameView/PlayerSymbol.vue

```diff
@@ -0,0 +1,65 @@
+<script setup lang="ts">
+import { computed } from "vue";
+import TaegeukStone from "../TaegeukStone.vue";
+
+const props = defineProps<{
+  variant: string;
+  config: unknown;
+  gamestate: unknown;
+  player_nr: number;
+}>();
+
+const colors = computed(() => {
```

Comment:
Could we push the color logic into the variants themselves? Benefits:

- baduk subclasses get their impl for free
- no need for extended if statements
- don't need to worry about Vue's reactivity

Something like this in AbstractGame:

```
playerColor(player: number): string[]
```

Reply:
My concern is variants where the move colors change from round to round. So far we don't have such a variant, but an example is the one I've been working on, sequential fractional.

I chose this general structure because I wanted to keep all the flexibility, i.e. enable variants to define their own UI component that gets displayed.

Reply:
> My concern is variants where the move colors change from round to round

Oh makes sense! Though I wasn't sure if SFractional might just use the primary color as the symbol.

> i.e. enable variants to define their own UI component that gets displayed.

Makes sense, I think this is a good feature to enable (#134).  Though I hadn't envisioned this inside the SeatComponents - for the Thue-Morse example, wouldn't we just want a standalone UI component?  Something like this:

![image](https://github.com/user-attachments/assets/508701fe-6ca9-4ad8-820f-054d5d791b33)

Reply:
Yeah I agree. So would it be ok if I add this to the Variant interface:

`  getPlayerColors: (config: ConfigT, playerNr: number) => string[];`

and then display a Stone with these colors as the player symbol?

Reply:
That works for me! And definitely seems like we can add gamestate later if needed for sfractional et al